nrm_django/grants/models.py

```diff
@@ -575,3 +575,15 @@ class Meta:
 
     # TO-DO: custom save?
     # * Create CN
+
+
+class Category(models.Model):
+    cn = models.CharField(primary_key=True, max_length=34)
+    grant = models.ForeignKey(Grant, models.DO_NOTHING, db_column="grant_cn")
+    category_cd = models.CharField(max_length=2)
```

Comment:
I'm guessing there are choices for these?  Since they're two character, I'm wondering if there may be a short and long version. If so, it may be defined in the User Guide.